src/libostree/ostree-repo-private.h

```diff
@@ -161,6 +164,8 @@ struct OstreeRepo {
   gchar *collection_id;
   gboolean add_remotes_config_dir; /* Add new remotes in remotes.d dir */
   gint lock_timeout_seconds;
+  guint64 payload_link_threshold;
+  int payload_fs_support_reflink; /* The underlying filesystem has support for ioctl (FICLONE..) */
```

Comment:
Minor bikeshed; how about just `fs_supports_reflink` ?  (It might be useful to know in another circumstance)

Reply:
yes, will rename it